### PR TITLE
Better error handling

### PR DESCRIPTION
**Pull request by koraa**
Improves error messages

@trieloff Can you check whether the change in openwhisk/static.js makes sense?

### CODE REVIEW

File: src/openwhisk/static.js [hunk 1]
```diff
@@ -129,13 +129,7 @@ function deliverPlain(owner, repo, ref, entry, root) {
...
       },
     };
-  }).catch((rqerror) => {
-    console.error('REQUEST FAILED', rqerror.response.body.toString());
-    if (rqerror.statusCode === 404) {
-      return error(rqerror.response.body.toString(), 404);
-    }
-    return error(rqerror.message, rqerror.statusCode);
-  });
+  }).catch(rqerror => error(rqerror.response.body.toString(), rqerror.statusCode));
```
Comment (trieloff) [inline src/openwhisk/static.js]: Always responding with `body.toString()` makes sense, but I'd differentiate between the actual status codes we are seeing from `raw.githubusercontent.com`:
- 404: is actually a 404
- 400 (when there is no file name, just a directory): should be reported as a 404
- other cases: just pass it through

**Comment by trieloff** [comment]
I'd also make sure to hit 100% diff coverage, w^H I got into this error by not testing enough.

**Comment by trieloff** [comment]
@koraa are you still working on this issue or should I take over?

**Comment by koraa** [comment]
Sorry for the late response!

@trieloff I think it makes sense for you to take over the PR since you wrote the code and probably know better what needs testing and what does not :)

**Comment by trieloff** [comment]
Will do.

**Pull request merged**